ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -59,6 +61,24 @@
   private static final String KEYBOARD_TYPE_NUMERIC = "numeric";
   private static final InputFilter[] EMPTY_FILTERS = new InputFilter[0];
 
+  private static final Map<String, Map<String, Integer>> TEXT_ALIGN_CONSTANTS;
+  static {
```

Comment:
you don't need a static block for this, you can just continue with this assignment on the line above